scalapact-http4s-0-23/src/main/scala/com/itv/scalapact/http4s23/impl/PactStubber.scala

```diff
@@ -0,0 +1,72 @@
+package com.itv.scalapact.http4s23.impl
+
+import cats.effect._
+import cats.effect.unsafe.implicits.global
+import com.itv.scalapact.shared.http.SslContextMap
+import com.itv.scalapact.shared.{IInteractionManager, IPactStubber, ScalaPactSettings}
+import com.itv.scalapact.shared.json.{IPactReader, IPactWriter}
+import org.http4s.blaze.server.BlazeServerBuilder
+
+import scala.concurrent.Future
+
+class PactStubber extends IPactStubber {
+
+  private var instance: Option[() => Future[Unit]] = None
+  private var _port: Option[Int]                = None
```

Comment:
The way this becomes a Some leads to a race condition. See line 57 of this file & line 61ish of ScalaPactMock. I don't know why it isn't a problem in the CE2 version.